### ISSUE TITLE
[feature]: Jira Importer Documentation.

### ISSUE THREAD

**Issue by Maxwell175** [makeplane/plane#2817]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Summary

I saw that there was an issue that was opened and closed a few months ago adding the Jira importer, but the documentation page (https://docs.plane.so/importers/jira) is still empty so there is no way for me to figure out how full featured it is without actually setting up an instance.

### Why should this be worked on?

Since we already have 100s or existing projects in Jira it is important for us to be able to carry forward all the data from these projects, so assessing the ability to migrate is essential to us.

**Comment by vihar** [comment]
@Maxwell175, thanks for flagging this. Are there any specific areas that require more focus in terms of docs? Alternatively, does something like our [github impoprter docs](https://docs.plane.so/importers/github) work?

**Comment by Maxwell175** [comment]
Yes. That style is very good! But since Jira has so much more features that need migration, I would expect that section to be a lot bigger. Like do custom statuses get migrated? Custom fields? The formats of the custom fields? How well are multiple projects handled?

**Comment by vihar** [comment]
Thank you, @Maxwell175, for providing the details. We now have a clear understanding of what needs to be included in the documentation.

Regarding custom fields, this is an upcoming feature. As of today, custom fields cannot be imported. However, we will include a note in the documentation about the ongoing progress of this feature.

**Comment by vihar** [comment]
The docs are written now: https://docs.plane.so/importers/jira

They cover what you asked about — custom fields, status and priority mapping, sprints as cycles, components as modules, story points, worklogs, comments, attachments, sub-tasks and linked issues — along with known limitations.

Closing this out.
